File: UIClient/src/main/java/cz/zelenikr/remotetouch/security/Hash.java
package cz.zelenikr.remotetouch.security;

/**
 * @author Roman Zelenik
 */
public interface Hash {
    /**
     * Makes hash from the specific string.
     *
     * @param text the given text you want to hash
     * @return hash value like a plain text
     */
    String hash(String text);
}

File: UIClient/src/main/kotlin/cz/zelenikr/remotetouch/security/Hash.kt
package cz.zelenikr.remotetouch.security

/**
 * @author Roman Zelenik
 */
interface Hash {
    /**
     * Makes hash from the specific string.
     *
     * @param text the given text you want to hash
     * @return hash value like a plain text
     */
    fun hash(text: String): String
}